edgelet/edgelet-http-workload/src/server/cert/server.rs

```diff
@@ -61,7 +61,11 @@ where
             })
             .map(|(module_id, genid)| {
                 let module_id = module_id.to_string();
-                let alias = format!("{}{}server", module_id, genid.to_string());
+                let alias = format!(
+                    "aziot-edge/module/{}:{}:server",
```

Comment:
I've been using the name `aziot-edged` for the edge module. So for consistency, this should be `aziot-edged` (or rename instances of `aziot-edged` to `aziot-edge`)

Reply:
Done.